app/eventyay/base/models/auth.py

```diff
@@ -227,6 +227,10 @@ class Meta:
             'administrator': is_administrator,
         }
 
+    @property
+    def name(self):
```

Comment:
If other code tries to access `User.name` and fail, that code should be fixed.
Don't alias `fullname` to `name` because the `fullname` is clearer.
If you use only `name`,  people will ask: Is it first name, nick name for full name.